templates/views/create.twig

```diff
@@ -303,6 +321,28 @@
     </div>
   </div>
 
+<!-- jQuery -->
```

Comment:
Let's not put these back. Looks like in missed in #18 review that the JS blocks weren't wrapped in `{% block page_footer %} .. {% endblock %}`. @romanguerrero Could you take a look?

Reply:
I asked Roman if he was going to fix this before I made the pull request and he said he was stuck and that I should just go ahead.